api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java

```diff
@@ -163,7 +153,10 @@ private boolean canNotPan()
     {
```

Comment:
Maybe rename to `canNotPanX()` to keep it consistent?

Reply:
I have commit asking change. I add document too on canPanY().